build.gradle

```diff
@@ -180,6 +180,15 @@ project('segmentstore:storage:impl') {
         testCompile group: 'org.apache.hadoop', name: 'hadoop-minicluster', version: hadoopVersion, withoutLog4j
         testCompile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
 
+        //For ECS
+        /*testCompile group: 'io.findify', name: 's3mock_2.12', version: '0.2.2'
+        testCompile group: 'com.sun.jersey', name: 'jersey-json', version: '1.19'*/
+        testCompile group: 'org.gaul', name: 's3proxy', version: '1.5.2'
+        
+
+        compile "com.emc.ecs:object-client:3.0.0"
+        compile "com.sun.jersey:jersey-core:1.19"
```

Comment:
I believe we use jersey elsewhere, is this addition compatible with other uses across Pravega?